src/main/java/de/embl/cba/bdv/ome/zarr/zarr/N5ZarrReader.java

```diff
@@ -55,11 +57,17 @@
  */
 public class N5ZarrReader extends N5FSReader
```

Comment:
Some general comments on this class and the overall design:
- This class includes logic that is special to ome.zarr, e.g. the `axesMap`, hence I think it should be called `N5OmeZarrReader`.
- What's the relation to the `OmeZarrReader`? Do we really need to have both classes?
- We should somewhere check that the `axes` are valid, i.e. that they are in one of the combinations that we support, e.g. `czyx`, `zyx`, etc. (Note that the ome.zarr spec is currently not very clear if something like `xyzc` could exist, so we should have explicit checks for axes combinations that are not supported by us).

Reply:
- I re named the class
- As for the code duplication:
`OmeZarrReader` is an analog to the `OMEZarrS3Reader`, but non-s3 one is for the file system.
`N5S3ZarrReader` has a lot in common with `N5OmeZarrReader` 
`N5ImageLoader` has a lot in common with `N5OMEZarrImageLoader`
But I think that we slowly are solving these problemms.

Reply:
Thanks for the clarification re the second point. Let's keep it separate for now and see if it's possible to merge it later.